lib/ZendCode/VarTag.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @package ZendCode
+ */
+namespace Wsdl2PhpGenerator\ZendCode;
+
+use Zend\Code\Generator\DocBlock\Tag\ParamTag;
+
+class VarTag extends ParamTag
```

Comment:
Not really sure this should be _here_. But there are multiple (ok, just two:) uses of this and I thought its better to have this, than not.